packages/kit/README.md

```diff
@@ -1,15 +1,15 @@
-# @sveltejs/kit
+# The fastest way to build Svelte apps
 
-Here be dragons, etc etc.
+This is the [SvelteKit](https://kit.svelte.dev) framework and CLI. 
```

Comment:
The trailing space is making linting fail.

If you're using VS Code and you have the appropriate extensions installed, then you _should_ already be getting format-on-save happening, and this sort of thing shouldn't be a problem.